src/Components/SearchBar.css

```diff
@@ -0,0 +1,13 @@
+.searchTextField {
```

Comment:
Why file name starting with Capital letters?

Reply:
https://hackernoon.com/structuring-projects-and-naming-components-in-react-1261b6e18d76